draft-irtf-cfrg-vdaf.md

```diff
@@ -4151,9 +4151,6 @@ elements.) The scheme is comprised of the following algorithms:
     * `nonce` MUST be of length `Idpf.NONCE_SIZE` and chosen uniformly at random by the Client (see
       {{nonce-requirements}}).
 
-  > TODO(issue #255) Decide whether to treat the public share as an opaque byte
-  > string or to replace it with an explicit type.
```

Comment:
I think it would be worthwhile to separate public share encoding/decoding from the sharding and preparation routines, and using a type other than `bytes` for the public share internally. This is now the last composite data structure we represent as a byte string in the API, so it would be good to clean this up for consistency.

Reply:
Done.